### DISCUSSION TITLE
Testing the Silk Platform in 2024: Achieving 20 GiB/s I/O Throughput in a Single Cloud VM | Tanel Poder Consulting

### DISCUSSION THREAD

**Discussion by giscus** [tanelpoder/blog-comments#49] [Announcements]
# Testing the Silk Platform in 2024: Achieving 20 GiB/s I/O Throughput in a Single Cloud VM | Tanel Poder Consulting

Hands-on technical analysis of a novel data platform for high-performance block I/O in the cloud, tested by Tanel Poder, a database consultant and a long-time computer performance nerd.
Index Background and motivation Scalable Architecture by Design Enterprise Features Testing Results I/O Latency 1.3 Million IOPS Lessons Learned Summary Background and Motivation Back in 2021, my old friend Chris Buckel (@flashdba) asked me if I would like to test out the Silk Data Platform in the cloud and see how far I could push it. - Linux, Oracle, SQL performance tuning and troubleshooting - consulting & training.

https://tanelpoder.com/posts/testing-the-silk-platform-in-2024/

**Comment by nickpclarke** [comment]
Fantastic work! Such an astounding readout.

**Reply by tanelpoder** [reply]
Thanks!!!
